### DISCUSSION TITLE
How to create a simple hello world with visualization

### DISCUSSION THREAD

**Discussion by yzd0014** [jrouwe/JoltPhysics#1185] [General]
After I generated the visual studio project, I found the provided hello world application doesn't have any visualization. Project samples have visualization, but it's not easy to create my own simple test cases. Any suggestions how I can create my own simple test cases with visualization?

**Comment by Mahmoud1205** [comment]
If you mean debug rendering here it is, https://jrouwe.github.io/JoltPhysics/index.html#debug-rendering
